saw-core-coq/src/Language/Coq/AST.hs

```diff
@@ -9,7 +9,16 @@ Portability : portable
 
 module Language.Coq.AST where
 
-type Ident = String
+import Data.String (IsString(..))
+
+newtype Ident = Ident String
```

Comment:
Are there any invariants about the underlying `String` (e.g., module structure) worth documenting in the Haddocks for this type?